cmd/devp2p/internal/ethtest/suite.go

```diff
@@ -78,10 +79,14 @@ func (s *Suite) EthTests() []utesting.Test {
 		// test transactions
 		{Name: "TestTransaction", Fn: s.TestTransaction},
 		{Name: "TestInvalidTxs", Fn: s.TestInvalidTxs},
-		{Name: "TestLargeTxRequest", Fn: s.TestLargeTxRequest},
 		{Name: "TestNewPooledTxs", Fn: s.TestNewPooledTxs},
 		{Name: "TestBlobViolations", Fn: s.TestBlobViolations},
 	}
+	// This test is often failing on 32-bit CI.
+	if bits.UintSize > 32 {
+		tests = append(tests, utesting.Test{Name: "TestLargeTxRequest", Fn: s.TestLargeTxRequest})
+	}
+	return tests
```

Comment:
I would prefer to  use different semantics. Instead of
"Are we running on 32-bit? If so, skip", the logic should be
"Are we running with the tag `short`? If so, skip".

Reply:
I.e
```golang
	if testing.Short() && test.Name == "TestLargeTxRequest"{
		t.Skip("skipped in short-mode")
	}
```

Reply:
Please take another look. I refactored so skipping is based both on `short` tag or 386 arch. This is similar to how we decide on skipping the state tests:

https://github.com/ethereum/go-ethereum/blob/ae4ea047e35bb35828231f1b93f2f65a964abdc9/tests/init_test.go#L144-L149